Makefile

```
@@ -24,7 +24,7 @@ install-dev-tools:  ## Installs all necessary cargo helpers
 	cargo install cargo-risczero
 	cargo risczero install
 
-lint:  ## cargo check and clippy. Skip clippy on guest code since it's not supported by risc0
+lint: check-fuzz  ## cargo check and clippy. Skip clippy on guest code since it's not supported by risc0
 	## fmt first, because it's the cheapest
 	cargo +nightly fmt --all --check
 	cargo check --all-targets --all-features
```

This would maintain the ordering from cheapest to most expensive check:

```suggestion
lint:  ## cargo check and clippy. Skip clippy on guest code since it's not supported by risc0
	## fmt first, because it's the cheapest
	cargo +nightly fmt --all --check
	cargo check --all-targets --all-features
	$(MAKE) check-fuzz
```